Promote/DO App Platform Example (#280)
1Password/scim-examples: PREPARATION.md

You'll need to create a DNS record with your SCIM bridge domain name after the bridge is deployed. Follow the steps in each respective deployment guide to finish setting up your DNS record at the appropriate time.

> **Note**
>

> >Container-as-a-Service platforms (for example, Azure Container Apps, DigitalOcean App Platform) include TLS endpoint management, so you do not need to create a DNS record for these deployments.

Reviewer: I think we can remove the hyphenation for CaaS and the capitalization here and in other instances. Also, it should be plural without "platforms". Usage elsewhere varies, but [Atlassian](https://www.atlassian.com/microservices/cloud-computing/containers-as-a-service) doesn't use hyphens and neither does [IBM](https://www.ibm.com/topics/containers-as-a-service) so I'd rather we stick to that. Right now it reads a little strange. I've also added "and" instead of a comma, and a contraction instead of "do not".

```suggestion
>Containers as a service, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to create a DNS record for these deployments.
```

Reviewer 2: I'm in agreement with the superfluous hyphens, but "Container as a Service platforms" reads much better and is more appropriate to me than "Containers as a Service". Azure Container Apps or DigitalOcean App Platform are each platforms that allow customers to host Containers as a Service.

Reviewer: I'm happy to use that instead :)

Author: Combining both of your suggestions here. 

```suggestion
>Containers as a service platforms, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to create a DNS record for these deployments.
```

>

### TLS certificate

Identity providers typically require a TLS certificate when communicating with the SCIM bridge. By default, TLS certificates are handled by a complimentary [Let's Encrypt](https://letsencrypt.org/) service integration, which automatically generates and renews a certificate based on the domain you're using.

> **Note**
>

> >Container-as-a-Service platforms (for example, Azure Container Apps, DigitalOcean App Platform) include TLS endpoint management, so you do not need to use our optional certificate manager component.

Reviewer: To match the changes above, but for this context I've removed the examples since they were just mentioned a few paragraphs before:

```suggestion
>Containers as a service, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to use the optional certificate manager component.
```

Reviewer 2: Same as similar comment above (prefer we keep the word platforms here).

Author: Combining above suggestions

```suggestion
>Containers as a service platforms, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to use the optional certificate manager component.
```

## Clone this repository

> For most custom deployments, it is recommended that you clone this repository before you start the deployment. This ensures you have all the files you need and provides an opportunity for you to familiarize yourself with the contents of the deployment method you've selected.

Reviewer: To be a bit more direct with the recommendation, we can use an active voice:

```suggestion
For most custom deployments, we recommend that you clone this repository before you start the deployment. This ensures you have all the files you need and provides an opportunity for you to familiarize yourself with the contents of the deployment method you've selected.
```
